test/SlowTests/Issues/RavenDB_15409.cs

```diff
@@ -15,12 +19,15 @@ public class RavenDB_15409 : ClusterTestBase
         [Fact]
         public async Task DoNotCallUpdateLicenseLimitsCommandOnEveryLeaderChange()
         {
+            await using var logStream = new MemoryStream();
+            LoggingSource.Instance.AttachPipeSink(logStream);
```

Comment:
Won't this cause us to also gather data from other tests? What if there are more locations for this?

Reply:
Yes, but it may give us  the information we need and after fixing the test/code we can remove it.